am-ui/src/app/components/ads/ads.component.ts

```diff
@@ -1,10 +1,81 @@
-import {Component} from '@angular/core';
+/* eslint-disable prettier/prettier */
```

Comment:
können wir die prettier rule anpassen für die imports oder warum ist es nötig?

Reply:
Nein ist für das ganze File :). Irgend wie hat es Probleme die ich mir nicht erklären kann. Daher habe ich es einfach disabeld. In der Pipeline würde es aber nur warnings geben und keine Fehler. Das Problem ist irgend etwas mit "Delete cr"